.github/workflows/deploy-cluster.yml

```diff
@@ -62,6 +62,17 @@ jobs:
         terraform_root_folder: templates/new_service/terraform/domains/environment_domains
         terrafile_environment: development
 
+    - name: Send Slack notification on failure
```

Comment:
This job runs on every PR. This would send an alert to the channel for any validation error. Can we add a check to only run when we're on the main branch?

Reply:
Fixed

Reply:
I can't see the condition sorry